definitions/npm/helmet_v3.x.x/flow_v0.38.x-/helmet_v3.x.x.js

```diff
@@ -0,0 +1,93 @@
+import type { Middleware, Request, Response } from 'express'
```

Comment:
This doesn't work. Sadly it will result in any.
Could should be able to import that within the module declaration though.

So could you move all the declares into the declare module?

Reply:
@marudor are you suggesting this TODO can be fixed now as well? Thanks.
https://github.com/flowtype/flow-typed/blob/master/definitions/npm/backbone_v1.x.x/flow_v0.20.x-/backbone_v1.x.x.js#L2